cli/tests/testdata/eval_context_throw_with_conflicting_source.ts.out

```diff
@@ -1,5 +0,0 @@
-[WILDCARD]error: Uncaught Error: foo
-Warning Couldn't format source line: Column 31 is out of bounds (source may have changed at runtime)
```

Comment:
This warning likely doesn't happen anymore with using the proc state cache, but I'm hesitant to remove the sanity checks/warnings because this was a panic that kept popping up in the past and there could be other edge cases where the source mapped stack frame isn't consistent with the code currently in the ps cache.

Reply:
I think it's needed - any file you run using `createRequire()` that throws an error will have it (because when you `require()` something it's wrapped in an IIFE which screws up lines because `deno_core` is not aware of that wrapping).